sklearn/linear_model/tests/test_base.py

```diff
@@ -205,6 +208,25 @@ def test_linear_regression_sparse_multiple_outcome(random_state=0):
     assert_array_almost_equal(np.vstack((y_pred, y_pred)).T, Y_pred, decimal=3)
 
 
+def test_linear_regression_pd_sparse_dataframe_warning():
+    try:
+        import pandas as pd
+        if LooseVersion(pd.__version__) < '0.24.0':
```

Comment:
Please include a comment stating this is because `is_sparse` is not available in older versions of pandas.

Reply:
Actually it's available there but it doesn't work properly.

Reply:
May you include a comment that states this so future maintainers know why we are restricting the pandas version?

Reply:
Yes, I have added that comment now. Tell me if any changes are required in that.